app/src/androidTest/java/org/simple/clinic/storage/migrations/MigrationHelper.kt

```diff
@@ -0,0 +1,133 @@
+package org.simple.clinic.storage.migrations
+
+import androidx.sqlite.db.SupportSQLiteDatabase
+import org.simple.clinic.assertValues
+import org.simple.clinic.insert
+import org.simple.clinic.patient.Gender
+import org.simple.clinic.patient.Patient
+import org.simple.clinic.patient.PatientAddress
+import org.simple.clinic.patient.PatientProfile
+import org.simple.clinic.patient.PatientStatus
+import org.simple.clinic.patient.ReminderConsent
+import org.simple.clinic.patient.SyncStatus
+import org.simple.clinic.patient.businessid.BusinessId
+import org.simple.clinic.patient.businessid.Identifier
+
+fun SupportSQLiteDatabase.savePatientProfile(patientProfile: PatientProfile) {
+  savePatientAddress(patientProfile, this)
+  savePatient(patientProfile, this)
+  savePatientBusinessIds(patientProfile, this)
+}
+
+private fun savePatientAddress(patientProfile: PatientProfile, database: SupportSQLiteDatabase) {
```

Comment:
These were intentionally left specific to the migration class. The reason being that the database fields which map to column names and which fields need to be written to are specific to a given database schema version and cannot be reused across migration tests since the schemas will be different in each one.

This is fine for now, but let's revert this change later.